app/controllers/api/v1/accounts/credentials_controller.rb

```diff
@@ -22,4 +23,20 @@ def update
   def account_params
     params.permit(:display_name, :note, :avatar, :header, :locked)
   end
+
+  def user_settings_params
+    return nil unless params.key?(:source)
+
+    source_params = params.require(:source).permit(
+      :privacy,
+      :sensitive
+    )
+    sensitive = source_params.key?(:sensitive) ? source_params[:sensitive] : object.user.setting_default_sensitive
+    ActionController::Parameters.new(
+      user: {
+        setting_default_privacy: source_params.key?(:privacy) ? source_params[:privacy] : object.user.setting_default_privacy,
+        setting_default_sensitive: sensitive ? '1' : '0', # database doesn't like boolean values for default_sensitive
```

Comment:
It's not database who doesn't like boolean value, but `UserSettingsDecorator` is. `UserSettingsDecorator` should be updated instead of adding a workaround.

Reply:
Ok, I can do that. I wasn't sure about the additional effects of making a change like that.